pkg/kubectl/cmd/top_node.go

```diff
@@ -40,6 +40,7 @@ type TopNodeOptions struct {
 	HeapsterOptions HeapsterTopOptions
 	Client          *metricsutil.HeapsterMetricsClient
 	Printer         *metricsutil.TopCmdPrinter
+	Sort            string
```

Comment:
It took me a second to figure out what this is (it reads like a boolean option). Maybe SortBy or SortName?